.github/workflows/phpstan.yml

```diff
@@ -60,13 +60,13 @@ jobs:
 
     # https://docs.github.com/en/free-pro-team@latest/actions/guides/about-service-containers
     services:
-      mysql:
+      database:
         image: ${{ matrix.db-image }}
         env:
           MYSQL_ROOT_PASSWORD: root
         ports:
           - 3306:3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+        options: ${{ startsWith(matrix.db-image, 'mariadb') && '--health-cmd="healthcheck.sh --connect --innodb_initialized" --health-interval=1s --health-timeout=10s --health-retries=60' || '--health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3' }}
```

Comment:
reverted to the variant I did not like at first, because its the easiest way for now to get the builds green - which is most important.

Reply:
May I disagree?
The most important is long-term sustainability.
Why fix an error today if you have to refix it 12 more times in the coming months??

Reply:
I had a red CI pipeline for mariadb for a few months and did not know why. thanks to your PR I have an idea how to fix, so I want at first fix the problem to get the pipeline green again, which is super important.

I am still open to the matrix refactoring PR you submitted